API/src/routes/Assessment/index.js

```diff
@@ -9,18 +9,25 @@ module.exports = server => {
     `${ BASE_URL }/submit`,
     async (req, res, next) => {
       try {
-        const { assessment } = req.params;
+        //original
+        // const { assessment } = req.params;
+        const  assessment  = req.params;
     
         //verify that your data is making it here to the API by using console.log(assessment);
+        console.log('in api router')
+        console.log(assessment);
+        
         //call the AssessmentService.submit function from the API/src/microservices/Assessment/ and supply the correct parameters
-    
+        const ben  = await AssessmentService.submit(assessment);
```

Comment:
```suggestion
        await AssessmentService.submit(assessment);
```
You aren't returning anything from this function, so you shouldn't need the variable.

Reply:
I am using 'ben' (which I will change to 'assessment') in the ResponseHandler component which eventually returns it to the front end. The ResponseHandler 
 is currently providing the front end with the 'assessment' data that was just added to the database, plus a success message. I agree that the front-end should not get the assessment data that was just added as it is probably not secure, and as it makes no use of it. 
But I was also using the 'ben' to see if the save() to the database worked. I did this by seeing if 'ben.attributes' was defined - I was not sure how to do that using the 'res' argument coming in to the ResponseHandler handler component.

Reply:
If it fails, the error should be picked up and passed to the front-end automatically by restify / express, so you don't have to do this yourself.

Reply:
Thanks, I changed this as recommended.